### PR TITLE
github: add new issue templates

### PR DESCRIPTION
**Pull request by manav2401**
This PR updates the github issue templates for bug, feature and question category. 

Checkout these files for preview: [Bug](https://github.com/maticnetwork/bor/blob/manav/POS-628-update-issue-template/.github/ISSUE_TEMPLATE/bug.md), [Feature](https://github.com/maticnetwork/bor/blob/manav/POS-628-update-issue-template/.github/ISSUE_TEMPLATE/feature.md) and [Question](https://github.com/maticnetwork/bor/blob/manav/POS-628-update-issue-template/.github/ISSUE_TEMPLATE/question.md). 

Jira: [POS 628 - Update github issue templates](https://polygon.atlassian.net/browse/POS-628)

### CODE REVIEW

**Review by pratikpatil024 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -1,30 +1,70 @@
...
+NOTE: Please make sure to check of any addresses / private keys / rpc url's / IP's before sharing the logs or anything from the additional information section (start.sh or heimdall config).
+-->
 
-#### Expected behaviour
+#### **System information**
 
+Bor client version: [e.g. v0.2.16]
 
-#### Actual behaviour
+Heimdall client version: [e.g. v0.2.10]
```
Comment (JekaMas) [inline .github/ISSUE_TEMPLATE/bug.md:19]: Could we add instructions how a user could check versions of services?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -1,30 +1,70 @@
...
+<!--
+It should be start.sh if you're using bor v0.2.x and bor.service if it's bor v0.3.x. Mention this file if you're facing any issues like unable to use some flag/s according to their expected behaviour.
+-->
+2. The result of `eth.syncing`, `admin.peers.length`, value of the `maxpeers` flag in start.sh, and bootnodes/static nodes (if any) is you're facing some syncing issue. 
+<!--
+You can get the above results by attaching to the IPC using the command `bor attach $BORDIR/bor.ipc` or `bor attach $DATADIR/bor.ipc` and running the mentioned commands. 
+Mention this if you're facing issues where bor keeps stalling and is not importing new blocks or making any progress. Adding chain configuration mentioned in the previous step would also be really helpful here.  
+-->
+3. Your `heimdall-config.toml` parameters (whose ideal location is `~/.heimdalld/config/`) for checking the ETH and BOR RPC url's, incase of issue with bor heimdall communication.
```
[the remark this answers is not in the document]
Comment (manav2401) [reply .github/ISSUE_TEMPLATE/bug.md]: Thanks, done!

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -1,30 +1,70 @@
...
+
+**Additional Information**
+
+In order to debug the issue faster, we would stongly encourage if you can provide some of the details mentioned below (whichever seems relevant to your issue)
+
+1. Your `start.sh` file or `bor.service`, if you're facing some peering issue or unable to use some service (like `http` endpoint) as expected. Moreover, if possible mention the chain configuration printed while starting the node which looks something like `Initialised chain configuration config="{ChainID: 137, ..., Engine: bor}"`
+<!--
+It should be start.sh if you're using bor v0.2.x and bor.service if it's bor v0.3.x. Mention this file if you're facing any issues like unable to use some flag/s according to their expected behaviour.
+-->
+2. The result of `eth.syncing`, `admin.peers.length`, value of the `maxpeers` flag in start.sh, and bootnodes/static nodes (if any) is you're facing some syncing issue.
```
[the remark this answers is not in the document]
Comment (manav2401) [reply .github/ISSUE_TEMPLATE/bug.md]: Thanks, done!

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (manav2401) [reply .github/ISSUE_TEMPLATE/bug.md]: Sure, done!

**Pull request merged**